test/common/http/conn_manager_utility_test.cc

```diff
@@ -68,7 +68,8 @@ TEST_F(ConnectionManagerUtilityTest, ShouldTraceRequest) {
         Http::AccessLog::ResponseFlag::NoHealthyUpstream,
         Http::AccessLog::ResponseFlag::UpstreamRequestTimeout,
         Http::AccessLog::ResponseFlag::UpstreamOverflow,
-        Http::AccessLog::ResponseFlag::FaultInjected, Http::AccessLog::ResponseFlag::NoRouteFound};
+        Http::AccessLog::ResponseFlag::DelayInjected, Http::AccessLog::ResponseFlag::FaultInjected,
```

Comment:
So, is the intent that you just want to see why calls didn't arrive at Envoy2 ?

Or is the intent that you want to verify some things when a fault was injected (manually or automatically) ? If its the latter, then you need to include both delay and FI.

If its former (for some monitoring purposes), then making FIs traceable makes sense. But on that front, I would still argue that your monitoring dashboard would start ringing the alarm bells when calls to Envoy2 are being delayed by several seconds. In this case, won't you wan't to know that it was caused by delay injection and not because of some real event ? 
 IOW, it would avoid the pain of having to investigate because you can write filters that will stop alerting you if the faults were caused by DI/FI. Make sense?